docs/learn-staking.md

```diff
@@ -176,27 +174,28 @@ It is planned to be close to 10% in the first year. This means that each validat
 
 **Blue line**:  inflation rate
 
-The above chart shows the inflation model of the network. Depending on the staking participation, the inflation rate will be dynamically changed to incentivize / disincentivize token holders to participate in staking. For instance, inflation would be 10% if 50% of DOTs are being staked to the network.
+The above chart shows the inflation model of the network. Depending on the staking participation, the inflation rate will be dynamically changed to incentivize / disincentivize token holders to participate in staking. For instance, inflation would be 10% if 50% of DOTs are staked in the network.
 
-Determining the ideal staking rate is not an easy task as the network requires enough DOTs to be staked to provide the security guarantees we want and to avoid illiquidity on the market.
+Determining the ideal staking rate is not an easy task as the network requires enough DOTs to be staked to provide the security guarantees, and we want and to avoid illiquidity on the market.
 
 For those who are interested in knowing more about the design of inflation model for the network, please see [here](https://research.web3.foundation/en/latest/polkadot/Token%20Economics/).
 
 ## Why stake?
 
-- 10% inflation/year when the network launches
-- 50% targeted active staking
-- ~20% annual return
+* 10% inflation/year when the network launches
+* 50% targeted active staking
+* ~20% annual return
 
 ## Why not stake?
 
-- Tokens will be locked for about 12 weeks
-- Punishment in case of validator found to be misbehaving
+* Tokens will be locked for about 12 weeks
+* Punishment in case of validator found to be misbehaving
 
 ## How many validators will Polkadot have?
 
 The plan is to start with somewhere between 50 to 100 open validator positions and open more gradually. The top bound on the number of validators has not been determined yet, but should only be limited by the bandwidth strain of the network due to frequent and voluminous peer-to-peer message passing. The estimate of the number of validators that Polkadot will have at maturity is around 1000.
 
 ## Resources
 
-- [How Nominated Proof of Stake will work in Polkadot](https://medium.com/web3foundation/how-nominated-proof-of-stake-will-work-in-polkadot-377d70c6bd43) - Blog post by Web3 Foundation researcher Alfonso Cevallos covering NPoS in Polkadot.
+* [How Nominated Proof of Stake will work in Polkadot](https://medium.com/web3foundation/how-nominated-proof-of-stake-will-work-in-polkadot-377d70c6bd43) - Blog post by Web3 Foundation researcher Alfonso Cevallos covering NPoS in Polkadot.
+* [Secure validator setup](https://hackmd.io/QSJlqjZpQBihEU_ojmtR8g)
```

Comment:
I think this hackmd is out-of-date.  Our latest secure validator setup is here: https://wiki.polkadot.network/docs/en/maintain-guides-secure-validator

Reply:
```suggestion
* [Secure validator setup](https://wiki.polkadot.network/docs/en/maintain-guides-secure-validator)
```